### ISSUE TITLE
[BUG] Not importing correct paths

### ISSUE THREAD

**Issue by reezarabara** [onicagroup/runway#349]
When deploying with runway 1.8.0, pre/post build/destroy hooks are not being imported correctly.

Steps to recreate using 
[runway-test1.zip](https://github.com/onicagroup/runway/files/4702987/runway-test1.zip):

```
pipenv install
make deploy
```
Results:
```
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 962, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'abc.hooks'; 'abc' is not a package
make: *** [deploy] Error 1
```

**Comment by ITProKyle** [comment]
While the provided example does not reproduce the issue we discussed privately yesterday, I think I have been able to reproduce it with this - [import-bug.zip](https://github.com/onicagroup/runway/files/4703401/import-bug.zip).

**Comment by troyready** [comment]
@reezarabara a fix for this should be available now in v1.8.1 -- please test and let us know if it's still giving you any trouble.

Thanks again for your help!
